### ISSUE TITLE
tflite on arduino nano 33 ble 

### ISSUE THREAD

**Issue by winstonjthinker** [tensorflow/flutter-tflite#259]
Didn't find op for builtin opcode 'FULLY_CONNECTED' version '1'. An older version of this builtin might be supported. Are you using Didn't find op for builtin opcode 'FULLY_CONNECTED' version '1' an old TFLite binary with a newer model?

Failed to get registration from op code FULLY_CONNECTED
 
Failed to allocate tensors.
07:31:13.216 -> Didn't find op for builtin opcode 'FULLY_CONNECTED' version '1'. An older version of this builtin might be supported. Are you using an old TFLite binary with a newer model?
07:31:13.216 -> 
07:31:13.216 -> Failed to get registration from op code FULLY_CONNECTED
07:31:13.216 ->  
07:31:13.216 -> Failed to allocate tensors.

**Comment by padmanabanSampath** [comment]
same error, happening to me as well Didn't find op for builtin opcode 'FULLY_CONNECTED' version '11'. An older version of this builtin might be supported. Are you using Didn't find op for builtin opcode 'FULLY_CONNECTED' version '11' an old TFLite binary with a newer model?, but the interpreter at collab and kaggle process it without any problem., I have implemented all cures stated in online including builtin ops while converting, etc but none resolved it if managed to resolve please let me know,

**Comment by PaulTR** [comment]
Since this is an Arduino question, I have to ask: are you somehow using Flutter _on_ the Arduino Nano? I haven't heard of that being possible, so any documentation you have on that would be awesome. Otherwise I'd expect you'd use [TFLite for Microcontrollers](https://ai.google.dev/edge/litert/microcontrollers/overview), which is a whole other library/project unrelated to this one.

**Comment by padmanabanSampath** [comment]
No, Sorry for posting in ardunio section, I am also facing the same issue in flutter android side, but in collab and kaggle i am able to successfully load and run inference to the tflite model, but here i am getting this error Didn't find op for builtin opcode 'FULLY_CONNECTED' version '11'. An older version of this builtin might be supported. Are you using Didn't find op for builtin opcode 'FULLY_CONNECTED' version '11' an old TFLite binary with a newer model?, the version used to convert the model is tf2.17
